migration/v0.31/apply/50-ingress-nginx.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+    log_info "upgrading ingress-nginx"
+    helmfile_upgrade sc 'app=ingress-nginx'
+
+    helmfile_upgrade wc 'app=ingress-nginx'
```

Comment:
Any reason why this is done here and not in the regular apply snippet? (Given that it is run as a regular upgrade)

Reply:
Added it as in case if i needed it. But i do not think it is needed. So as you say i can just let the normal upgrade do the work.